sample/src/main/java/org/aerogear/graphqlandroid/activities/MainActivity.kt

```diff
@@ -113,16 +111,69 @@ class MainActivity : AppCompatActivity() {
         recycler_view.layoutManager = LinearLayoutManager(this)
         recycler_view.adapter = taskAdapter
 
+        //Used for in-memory mutations
         buttonOffline.setOnClickListener {
 
             Log.e(TAG, "buttonOffline clicked")
+            OfflineMutationSender()
+        }
 
-//            WorkManager.getInstance().enqueue(periodicWorkRequest)
-//            WorkManager.getInstance().enqueue(oneTimeWorkRequest)
+        //Used for persistent mutations
+        persistence.setOnClickListener {
 
-            OfflineMutationSender()
+            Log.e(TAG, "in persistence")
+
+            val listOfMutations = dbDao.getAllMutations()
+            Log.e(TAG, "in persistence size of list : ${listOfMutations.size}")
+
+            val client = OkHttpClient.Builder()
+                .addInterceptor(LoggingInterceptor()).build()
+
+            listOfMutations.forEach {
+                val mutationnum = it
+
+                val valuemapasjsonn = mutationnum.valuemap
+
+                val variables = valuemapasjsonn.toString()
+
+                val querydoc = mutationnum.queryDoc
+
+                val operationName = mutationnum.operationName.name()
+
+                val operationID = mutationnum.operationID
+
+                val stringQueryDoc: String = querydoc.replace("\n", " ")
+
+                val jsonMediaType: MediaType = MediaType.get("application/json")
+
+                val body: RequestBody = RequestBody.create(
+                    jsonMediaType, "{\"operationName\":" + "\"" + operationName + "\"" + "," +
+                            "\"variables\":" + variables + "," +
+                            "\"extensions\":{\"persistedQuery\":{\"version\":1,\"sha256Hash\":" + "\"" + operationID + "\"" + "}}," +
```

Comment:
Why is the version hard-coded to 1?

Reply:
Because when I was passing the correct version too, it was showing ` Unsupported persisted query version`